src/lib/utils.c

```diff
@@ -98,8 +98,11 @@ uint64_t get_uint64(const char *path, uint64_t defval, const char *name)
 	if (!p)
 		return defval;
 
-	str_toul(&defval, p, NULL, 16);
-	return defval;
+	errno = 0;
+	uint64_t t = strtoull(p, NULL, 16);
```

Comment:
there are reasons why we wrapped `strtol family` by macros. I don't remember right now exacly why we did this (and commit message is not helpful) but I would like to ask you to follow it or at least mention why you cannot use it.

https://github.com/intel/ledmon/commit/598bc0bfd7c7c40c3d825e12ec2cb477bab992f8

Reply:
There are no new static code analysis issues detected by our internal utils for now.

Reply:
The macro wrappers appear to have range checking for each different integer type.  At the moment, I don't see a benefit at for checking that a `uint64_t` is between 0 and UINT64_MAX.  The current macro implementation will not support uint64_t types.

I'll add more information to commit message.